### ISSUE TITLE
Exception in thread "main" bsh.Parser$LookaheadSuccess

### ISSUE THREAD

**Issue by sourceforge-issue-exporter** [beanshell/beanshell#361]
&gt; cat t6.java
```java
public class t6 {
  static StringBuffer $0 = new StringBuffer("Nigel is rubbish");
  static {
    $0.replace(9,16,"best");
  }
  public static void main(java.lang.String Z[]) {
    System.out.println($0);
    return;
  }
}
```
Process cat exited with code 0

Using Java produces the expected:
&gt; javac t6.java
Process javac exited with code 0
&gt; java t6
Nigel is best
Process java exited with code 0

Using an old BeanShell produces:
&gt; java -classpath ../bsh-2.0b4.jar bsh.Interpreter t6.java
Evaluation Error: Parse error at line 3, column 9.  
Encountered: static
Process java exited with code 0

The latest and bestest has changed but has it improved?
&gt; java -classpath ../bsh-2.0b5.jar bsh.Interpreter t6.java
Exception in thread "main" 
bsh.Parser$LookaheadSuccess
at bsh.Parser.&lt;init&gt;\(Parser.java:5943\)
at bsh.Interpreter.&lt;init&gt;\(Interpreter.java:175\)
at bsh.Interpreter.eval\(Interpreter.java:623\)
at bsh.Interpreter.source\(Interpreter.java:571\)
at bsh.Interpreter.main\(Interpreter.java:360\)
Process java exited with code 1

Reported by: nigel_galloway

Original Ticket: "beanshell/bugs/223":https://sourceforge.net/p/beanshell/bugs/223

**Comment by nickl-** [comment]
Feature request: implement static initialization block

[t6.java.txt](https://github.com/beanshell/beanshell/files/10145501/t6.java.txt)

**Comment by nickl-** [comment]
Closed: duplicate #663
